webapp/channels/src/components/admin_console/admin_console.test.tsx

```diff
@@ -102,8 +103,8 @@ describe('components/AdminConsole', () => {
             consoleAccess: {read: {}, write: {}},
             team: {name: 'development'} as Team,
         };
-        const wrapper = shallow(
-            <AdminConsole {...props}/>,
+        const wrapper = renderWithIntl(
+            <BrowserRouter><AdminConsole {...props}/></BrowserRouter>,
```

Comment:
I think the `currentUserHasAnAdminRole` props here should be `true` for this component to generate routes, and likely needs to render with `renderWithIntlAndStore`.